tests/interview-constants.js

```diff
@@ -2,7 +2,8 @@ const putils = require('./puppeteer-utils');
 
 // Each repo has its own interview that it's testing
 // We'll deal with multiple urls when it becomes an issue
-const INTERVIEW_URL = `${putils.BASE_INTERVIEW_URL}%3A209a_package.yml`;
+let filename = '209a_package';  // Easier to edit
```

Comment:
Should this be `encodeURIComponent(209a_package)`? Doesn't really matter for this filename, but it will for more exotic filenames with unusual characters.

Reply:
Probably to be thorough. Will add this as an issue. At least this PR doesn't make it worse than what we have now.